dev-lang/zig/zig-0.4.0-r1.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=6
```

Comment:
How is using EAPI 6 as opposed to 7 an improvement?

Reply:
EAPI=7 sets `BUILD_SHARED_LIBS=ON`, which causes the build to fail. I discussed this with @juippis and he suggested to set `EAPI=6` instead of overriding EAPI7's default setting.

Reply:
For now, yes. The current ebuild in the tree is in a bad shape and builds static libraries, this was the fastest most acceptable solution for now. 
For the next version bump, EAPI-7 is expected. Upstream just handles it weirdly, you need to study their CMake build files very carefully ;)
Still waiting on ack for this one from the higher-ups...

Reply:
Sounds good to me, thanks for explaining :)